impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -362,6 +362,29 @@ public void cacheSerializingCopiers() throws Exception {
   }
 
   @Test
+  public void cacheCopierAtCacheManagerLevel() {
```

Comment:
Should this test be here in `GettingStarted`? Since this test is not being referred in the getting started document, may be we could move this to the `CacheCopierTest` class in the *integration-test* module which was specifically created for all the copier related tests.